packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -521,6 +521,67 @@ const DatePickerInput = (props) => {
     }
   };
 
+
+
+  const handleInputKeydown = (event, inputType) => {
+    const { day, month, year } = date;
```

Comment:
Where is date defined?

Reply:
https://github.com/cerner/terra-framework/blob/5a4b1cf7be14410bd8173b971135be0edbba2ff9/packages/terra-date-picker/src/DateInput.jsx#L171